### PR TITLE
Bug 1370594: Beta and launch notices

### PR DESCRIPTION
**Pull request by stephaniehobson**
- add beta and launch notices, hidden by default
- add js to show notices on first visit to new design
- depends on a variable added in the header/footer/type PR

### CODE REVIEW

File: jinja2/base.html [hunk 1]
```diff
@@ -210,6 +210,46 @@
...
+        <p>MDN is changing to focus just on documenting web technologies! All the same great content is still here; we’re just changing some visual elements and navigation, to help you more quickly find the web technology docs you’re looking for.</p>
+        <p>But don’t worry, MDN and Mozilla are still together! In fact, we're updating MDN’s look to reflect Mozilla's new logo and colors.</p>
+      {% endtrans %}
+      <p><span class="redesign-old-logo"></span><span class="redesign-new-logo"></span></p>
+      {% trans blog_url='https://blog.mozilla.org/opendesign/future-mdn-focus-web-docs/' %}
+        <p><a href="{{ blog_url }}">Read more about the redesign</a> in our blog post. Thanks for using MDN!</p>
+      {% endtrans %}
+      <button id="redesignHide" type="button" class="only-icon transparent redesign-hide"><span>{{ _('Hide redesign announcement') }}</span><i aria-hidden="true" class="icon-times"></i></button>
+    </div>
+    {% endif %}
```
Comment (schalkneethling) [inline jinja2/base.html]: Nit: Indentation

File: jinja2/base.html [hunk 2]
```diff
@@ -210,6 +210,46 @@
   {% block content %}{% endblock %}
   </div></main>
 
+  <!-- redesign notices -->
```
Comment (jwhitlock) [inline jinja2/base.html]: Nit: move this block to an include, like ``jinja2/includes/redesign_notice.html``, and use ``{% include "includes/redesign_notice.html" %}``

File: kuma/static/js/redesign.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            var seenNotice = localStorage.getItem('redesignNotice') === 'true' ? true : false;
+            if (!seenNotice) {
+                var $redesignNotice = $(redesignNotice);
+
+                // wire close button
+                $('#redesignHide').on('click', function() {
+                    $redesignNotice.remove();
+                });
+                // show message
+                $redesignNotice.prependTo($('#content'));
```
Comment (jwhitlock) [inline kuma/static/js/redesign.js]: JS is magic 🎩 ✨. I had some problems getting this to work on the homepage, but it appears to be related to deleting ``redesignNotice`` and successfully refreshing.

File: jinja2/base.html [hunk 4]
```diff
@@ -210,6 +210,46 @@
...
+      {% trans blog_url='https://blog.mozilla.org/opendesign/future-mdn-focus-web-docs/', bug_url='https://bugzilla.mozilla.org/form.mdn#h=detail%7Cbug' %}
+        <h2>Under Construction</h2>
+        <p>Thanks for helping us test the <a href="{{ blog_url }}">new MDN look</a>! Find anything wrong? Please <a href="{{ bug_url }}">file a bug</a>. </p>
+      {% endtrans %}
+    </div>
+  </div>
+  <div id="redesignNotice" class="redesign-notice">
+    {% trans blog_url='https://blog.mozilla.org/opendesign/future-mdn-focus-web-docs/', profile_url=url('users.user_edit', username=user.username) %}
+      <h2>Hello Beta Tester!</h2>
+      <p>We’re updating the look and navigation of MDN to more closely reflect our dedication to providing the best (and most platform-agnostic) web docs in the world! As a beta tester, you’re the first to see the new look, and we’d love to hear what you think! </p>
```
Comment (stephaniehobson) [inline jinja2/base.html]: Oh good, it's not just me. Maybe I will remove one or two ;)

**Comment by jwhitlock** [comment]
Let's merge and do some practical testing in staging.

**Pull request merged**